pulumi/infra/emitter.py

```diff
@@ -24,6 +25,59 @@ def __init__(
             logical_bucket_name, sse=True, opts=pulumi.ResourceOptions(parent=self)
         )
 
+        # TODO: Unclear if this should be attached to our Bucket
+        # abstraction or here. Will keep it here for the time being.
+        #
+        # One nice thing about keeping it here is that we aren't
+        # creating policies for buckets that won't need them.
+        self.bucket_object_read_policy = aws.iam.Policy(
```

Comment:
Wouldn't `grant_get_to` work for this?

Reply:
It's the same permissions, yes, but that is method is attached to a bucket directly, whereas this is attached to an EventEmitter (which I now think needs to be recast as a kind of "notifying bucket"). The former is also a `RolePolicy` while this is a `Policy`. 

When we end up reviewing the IAM roles and policies, I think these two definitely need to be harmonized, but I'd like to hold off on this for now, since it's a bit of a bigger change.